### PR TITLE
Introduce 7.0 server version for 2024R1

### PR DESCRIPTION
**Pull request by guptadivya1501**

### CODE REVIEW

**Review by PProfizi — COMMENTED** [review]
Hi @guptadivya1501,
This will have to wait for the release of at least pydpf-core 0.8.1, plus the update to `ansys-dpf-server-2024-1-pre0` on ansys-dpf/dpf-standalone repository.

File: src/ansys/dpf/core/_version.py [hunk 1]
```diff
@@ -1,6 +1,6 @@
 """Version for ansys-dpf-core"""
 # major, minor, patch
-version_info = 0, 8, 1, "dev0"
+version_info = 0, 9, 0, "dev0"
```
Comment (anslpa) [inline src/ansys/dpf/core/_version.py:3]: @PProfizi is that ok?

Comment (PProfizi) [reply src/ansys/dpf/core/_version.py:3]: @anslpa, sure, meaning that our pydpf-core libraries developed for 2024R1 will be at least 0.9.0.

**Review by anslpa — APPROVED** [review]

File: src/ansys/dpf/core/_version.py [hunk 2]
```diff
@@ -23,5 +24,5 @@
     "5.0": "2023R1",
     "6.0": "2023R2",
     "6.1": "2023R2",
-    "6.2": "2023R2",
+    "7.0": "2024R1",
```
Comment (PProfizi) [inline src/ansys/dpf/core/_version.py:27]:
```suggestion
    "6.2": "2023R2",
    "7.0": "2024R1",
```

File: pyproject.toml [hunk 3]
```diff
@@ -5,7 +5,7 @@ build-backend = "flit_core.buildapi"
 [project]
 # Check https://flit.readthedocs.io/en/latest/pyproject_toml.html for all available sections
 name = "ansys-dpf-core"
-version = "0.8.2.dev0"
+version = "0.9.0.dev0"
```
Comment (PProfizi) [inline pyproject.toml:8]:
```suggestion
version = "0.8.2.dev0"
```

**Review by PProfizi — APPROVED** [review]

**Review by anslpa — APPROVED** [review]

**Pull request merged**